WrightTools/collection/_collection.py

```diff
@@ -102,6 +102,17 @@ def create_collection(self, name="collection", position=None, **kwargs):
         setattr(self, name, collection)
         return collection
 
+    def convert(self, units):
+        """Convert units of a collection.
+        Parameters
+        ----------
+        units: string
+            Units to convert to.
+        """
+        self.convert(units)
```

Comment:
This will be infinite recursion right off the bat.

Reply:
Yes... guess this is a placeholder for further discussion.